### ISSUE TITLE
wcseo:get-kvp-req63-66 because of missing attribute number-Returned attribute

### ISSUE THREAD

**Issue by lgoltz** [opengeospatial/ets-wcs20#58]
[FAILURE] The response to a successful DescribeEOCoverageSet request shall report in its number-Returned attribute the sum of all CoverageDescription and DatasetSeriesDescription elements included in the response

Assertions:
```
<xsl:if test="not(count($DescribeEOCoverageSet//*[local-name()='CoverageDescriptions'])+count($DescribeEOCoverageSet//*[local-name()='DatasetSeriesDescription'])=$DescribeEOCoverageSet//@*[local-name()='number-Returned'])">
  <ctl:message>[FAILURE] The response to a successful DescribeEOCoverageSet request shall report in its number-Returned attribute the sum of all CoverageDescription and DatasetSeriesDescription elements included in the response</ctl:message>
  <ctl:fail/>
</xsl:if>
<xsl:if test="$DescribeEOCoverageSet//@*[local-name()='number-Matched'] &lt; $DescribeEOCoverageSet//@*[local-name()='number-Returned']">
  <ctl:message>[FAILURE] The number-Returned shall be less than the number-matched</ctl:message>
  <ctl:fail/>
</xsl:if>
```

In http://schemas.opengis.net/wcs/wcseo/1.0/wcsEODescribeEOCoverageSet.xsd the attributes are named `numberMatched` and `numberReturned`.

**Comment by lgoltz** [comment]
Fixed xpath expression in PR #59 . Besides the spelling of the attributes `CoverageDescriptions` was changed to `CoverageDescription`.

**Comment by dstenger** [comment]
Successfully tested.

Test wcseo:get-kvp-req63-66 is passed.
